azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/manager/AudioSessionManager.kt

```diff
@@ -33,6 +33,9 @@ internal class AudioSessionManager(
             FeatureFlags.BluetoothAudio.active &&
                 (bluetoothAudioProxy?.connectedDevices?.size ?: 0 > 0)
 
+    private val bluetoothDeviceName
+        get() = bluetoothAudioProxy?.connectedDevices?.firstOrNull()?.name ?: ""
```

Comment:
Should we use context.getString(R.string.azure_communication_ui_setup_audio_device_bluetooth) here instead of empty string. This will allow us not to checks in the views for empty string.

Reply:
Dropped getString in the view, push the device name to Redux from AudioDeviceManager now, which is pulled into the VM.